tests/api/reports-performance-indicators.php

```diff
@@ -180,4 +190,116 @@ public function test_indicators_schema_allowed() {
 		$this->assertArrayHasKey( 'chart', $properties );
 		$this->assertArrayHasKey( 'label', $properties );
 	}
+
+	/**
+	 * Test the ability to aggregate Jetpack stats based on before and after dates.
+	 */
+	public function test_jetpack_stats_query_args() {
```

Comment:
How about a test for the default args if before/after are not supplied?

Reply:
Great idea, Paul.  Added in https://github.com/woocommerce/woocommerce-admin/pull/4291/commits/2424f50d7d6c376606b362278c1888cb9197a0b4